src/ontology/cl-edit.owl

```diff
@@ -29486,6 +29492,49 @@ SubClassOf(obo:CL_4023122 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023122 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001635))
 SubClassOf(obo:CL_4023122 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
 
+# Class: obo:CL_4023123 (hypothalamus kisspeptin neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30042117"^^xsd:string) obo:IAO_0000115 obo:CL_4023123 "A kisspeptin neuron that is located in the hypothalamus. These neurons project to and activate gonadotrophin-releasing hormone neurons (acting via the kisspeptin receptor) in the hypothalamus and stimulate the secretion of gonadotrophin-releasing hormone.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023123 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023123 <http://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(rdfs:label obo:CL_4023123 "hypothalamus kisspeptin neuron")
+EquivalentClasses(obo:CL_4023123 ObjectIntersectionOf(obo:CL_4023130 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001898)))
```

Comment:
This term is inferred as subclass of CARO_0000000, is that expected?

Reply:
I think thats due to the 'part of' some 'nervous system' which is an 'anatomical entity' (CARO_0000000) don't think it is a problem right? @dosumis